CHANGELOG.md

```diff
@@ -1,5 +1,150 @@
 # Change Log
 
+## v5.12.0
+
+## :tada: Features
+
+-   [feature] Configure format from file (#4155)
+-   [feature] Make cli expect array for rules dir (#3788)
+-   [feature] Set js rules to all valid active rules (#3641)
+-   [improvement] Add tests for typescript@2.9 and typescript@3.0 (#4053)
+-   [new-fixer][`type-literal-delimiter`](https://palantir.github.io/tslint/rules/type-literal-delimiter/) (#3964)
+-   [new-rule-option] Add whitespace rule for open braces
+-   [new-rule-option][`file-name-casing`](https://palantir.github.io/tslint/rules/file-name-casing/): Add file name whitelist (#4206)
+-   [new-rule-option][`object-literal-sort-keys`](https://palantir.github.io/tslint/rules/object-literal-sort-keys/): `locale-compare` (#4193)
+-   [new-rule-option][`strict-boolean-expressions`](https://palantir.github.io/tslint/rules/strict-boolean-expressions/) adds `allow-enum` to allow using enums in boolean expressions (#3604)
+-   [new-rule-option][`import-blacklist`](https://palantir.github.io/tslint/rules/import-blacklist/): support blacklisting specific named exports (#3926)
+-   [new-rule-option][`type-literal-delimiter`](https://palantir.github.io/tslint/rules/type-literal-delimiter/): Add `{ singleLine: "always" | "never"}` (#3964)
+-   [new-rule-option][`file-name-casing`](https://palantir.github.io/tslint/rules/file-name-casing/): Add `snake-case` option (#4081)
+-   [new-rule][`file-header`](https://palantir.github.io/tslint/rules/file-header/) (#3740)
+-   [new-rule][`ban-ts-ignore`](https://palantir.github.io/tslint/rules/ban-ts-ignore/) (#3728)
+-   [new-rule][`function-constructor`](https://palantir.github.io/tslint/rules/function-constructor/) (#4198)
+-   [new-rule][`increment-decrement`](https://palantir.github.io/tslint/rules/increment-decrement/) (#3575)
+-   [new-rule][`no-default-import`](https://palantir.github.io/tslint/rules/no-default-import/) (#4023)
+-   [new-rule][`no-unnecessary-constructor`](https://palantir.github.io/tslint/rules/no-unnecessary-constructor/) (#3647)
```

Comment:
Sorry, I renamed it to `unnecessary-constructor` during the PR and forgot to update the changelog entry ☹